### PR TITLE
Updated Kogito extension description and labels

### PR DESCRIPTION
**Pull request by mswiderski**

### CODE REVIEW

**Comment by mswiderski** [comment]
@emmanuelbernard here is the PR we talked about ... though I cannot set any labels myself

File: extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ---
 name: "Kogito"
+description: "Add business automation capabilities - processes and rules with Kogito (a toolkit that originates from projects Drools and jBPM)."
```
Comment (gastaldi) [inline extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: Usually we put the description in the pom.xml. Can you update https://github.com/quarkusio/quarkus/blob/master/extensions/kogito/runtime/pom.xml#L14 instead?

The keywords are fine

**Review by gastaldi — CHANGES_REQUESTED** [review]

File: extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml (hunk 1 above)
Comment (gastaldi) [reply extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: Sorry to not have it documented somewhere, but we're working on this :wink:

**Comment by emmanuelbernard** [comment]
You can define labels as

Keywords:
- foo
- bar

Check out
https://github.com/quarkusio/quarkus/blob/master/extensions/hibernate-orm/runtime/src/main/resources/META-INF/quarkus-extension.yaml#L5

File: extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml (hunk 1 above)
Comment (mswiderski) [reply extensions/kogito/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: description moved to pom.xml

**Comment by mswiderski** [comment]
@emmanuelbernard that's exactly what I did, added new labels to provide more options to find it.

**Review by gastaldi — APPROVED** [review]
Excellent! Thank you!

**Comment by gwenneg** [comment]
The test failure is unrelated, I'm relaunching the build.

**Pull request merged**